src/main/java/org/culturemedia/controllers/CultureMediaController.java

```diff
@@ -0,0 +1,26 @@
+package org.culturemedia.controllers;
+
+import java.util.*;
+
+import org.culturemedia.exception.VideoNotFoundException;
+import org.culturemedia.model.Video;
+import org.culturemedia.service.impl.CulturemediaServiceImpl;
+
+public class CultureMediaController {
+
+	private final CulturemediaServiceImpl cultureMediaService;
```

Comment:
Make the class more flexible by having CultureMediaController depend on the CulturemediaService interface.